shared/wallets/confirm-form/index.js

```diff
@@ -22,13 +21,6 @@ type ConfirmSendProps = {|
   bannerBackground?: Background,
   bannerText?: string,
   waitingKey?: string,
-  // TODO: these props are probably one level too high, and should be in Participants' container.
```

Comment:
when you do a `TODO` you wrote last week
![success_baby](https://user-images.githubusercontent.com/5677971/44484619-4616e200-a61c-11e8-99c6-0534dbd5b0b9.jpg)